Gemfile

```
@@ -10,4 +11,5 @@ group :jekyll_plugins do
     gem 'jemoji'
     gem 'unicode_utils'
     gem 'webrick'
+    gem 'jekyll-archives'
```

nit: please make sure all gems are listed in alphabetical order.

---

I'm not able to build the website without `jekyll-sitemap` included in the Gemfile.

```suggestion
    gem 'jekyll-archives'
    gem 'jekyll-sitemap'
```